HelpSource/Classes/Pipe.schelp

```diff
@@ -16,9 +16,17 @@ A link::Classes/String:: representing a valid shell command.
 argument::mode
 A link::Classes/String:: representing the mode. Valid modes are "w" (pipe to stdin) and "r" (pipe from stdout).
 
+method::argv
+
+argument::executableAndArguments
+A link::Classes/SequenceableCollection:: containining strings where the first string is the absolute path to the executable to be run and all other strings are passed as arguments to the executable. This method starts the process directly without using a shell.
```

Comment:
the "absolute path" part isn't required anymore after PATH support was added to `sc_popen_argv`, correct?